.github/workflows/publish.yml

```diff
@@ -28,26 +28,29 @@ jobs:
         uses: gittools/actions/gitversion/setup@v0.10.2
         with:
           versionSpec: "5.x"
+      - name: Setup Docker buildx
+        uses: docker/setup-buildx-action@v2.5.0
       - name: Determine Version
         id: gitversion
         uses: gittools/actions/gitversion/execute@v0.10.2
       - name: Extract metadata (tags, labels) for Docker
         id: meta
         uses: docker/metadata-action@v4
         with:
-          images: ${{ env.REGISTRY }}/brixel/singer
+          images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
           tags: type=semver,pattern={{version}},value=${{ steps.gitversion.outputs.semVer }}
       - name: Setup .NET
         uses: actions/setup-dotnet@v3
         with:
           dotnet-version: 6.x
       - name: Publish
         run: dotnet publish ./Singer.API/ --os linux --arch x64 -c Release /p:ContainerImageName=${{ env.IMAGE_NAME }} /p:ContainerImageTag=${{ steps.gitversion.outputs.semVer }}
-      - name: Tag built container with Github Container Registry url
-        uses: actions-hub/docker/cli@master
+      - name: Build and push Docker image
+        id: build-and-push
+        uses: docker/build-push-action@v4
```

Comment:
We don't need build and push since `dotnet publish` is already doing the docker build part for us

Reply:
Yes, I've noted it. Searching on a way to only do the push right now.